### PR TITLE
Completely remove Crypto.cpp and Crypto.hpp

### PR DESCRIPTION
**Pull request by yontyon**

### CODE REVIEW

File: kvbc/tools/db_editor/include/kv_blockchain_db_editor.hpp [hunk 1]
```diff
@@ -19,13 +19,18 @@
...
-#include "Crypto.hpp"
 #include "kvbc_key_types.h"
 #include "categorization/db_categories.h"
 #include "storage/merkle_tree_key_manipulator.h"
 #include "bcstatetransfer/DBDataStore.hpp"
 #include "bftengine/PersistentStorageImp.hpp"
 #include "bftengine/DbMetadataStorage.hpp"
+#include "crypto_utils.hpp"
+
+#pragma GCC diagnostic push
```
Comment (toly-kournik) [inline kvbc/tools/db_editor/include/kv_blockchain_db_editor.hpp:30]: shouldn't we do it in crypto_utils.hpp?

File: bftengine/src/bftengine/Digest.cpp [hunk 2]
```diff
@@ -13,11 +13,87 @@
...
+#elif defined SHA256_DIGEST
+#define DigestType SHA256
+#elif defined SHA512_DIGEST
+#define DigestType SHA512
+#endif
 
 namespace bftEngine {
 namespace impl {
 
+size_t DigestUtil::digestLength() { return DigestType::DIGESTSIZE; }
```
Comment (toly-kournik) [inline bftengine/src/bftengine/Digest.cpp:44]: Should it be a part of crypto_utils?

File: kvbc/tools/db_editor/include/kv_blockchain_db_editor.hpp (hunk 1 above)
Comment (yontyon) [reply kvbc/tools/db_editor/include/kv_blockchain_db_editor.hpp:30]: no, crypto_uils.hpp is agnostic to the implementation. The implementation is in crypto_utils.cpp

File: bftengine/src/bftengine/Digest.cpp (hunk 2 above)
Comment (yontyon) [reply bftengine/src/bftengine/Digest.cpp:44]: It is used only by bftengine, so I would prefer not to touch it now

File: kvbc/tools/db_editor/include/kv_blockchain_db_editor.hpp (hunk 1 above)
Comment (toly-kournik) [reply kvbc/tools/db_editor/include/kv_blockchain_db_editor.hpp:30]: so why is there include for crypto/dll.h?

**Review by toly-kournik — APPROVED** [review]
LGTM! Great refactoring

**Pull request merged**